### ISSUE TITLE
[Feature Request] Add a way to launch a game immediately on program launch through command line

### ISSUE THREAD

**Issue by TheNamlessGuy** [Ryujinx/Ryujinx#5982]
### Overview

I have a habit of keeping all my installed games as shortcuts on my desktop. While it certainly suffices to have a Ryujinx shortcut, it would be even better if I could have individual shortcuts for individual games.

This is a very low-priority, QoL type of thing.  
Also, sorry if this is already implemented - I couldn't find any documentation of it, and [the source code](https://github.com/Ryujinx/Ryujinx/blob/master/src/Ryujinx.Ui.Common/Helper/CommandLineState.cs) I found didn't seem to imply that this was possible currently.

### Smaller details

Perhaps a command line flag where you point to the game file location? `--launch C:/Games/MyGame` for example. That way you could create one bat-file per game, and shortcut those to the desktop.

### Nature of request

Not sure what I'm supposed to write here

### Why would this feature be useful?

For me, it's the reason given above (desktop shortcuts for individual games). I'm sure there are further ways to use it, but that's the one I'd like.

**Comment by MutantAura** [comment]
You should be able to right click any title in your game list and then select the "Create application shortcut" button.

**Comment by TheNamlessGuy** [comment]
> You should be able to right click any title in your game list and then select the "Create application shortcut" button.

Not sure how I never noticed that.  
I'll leave this open, but rephrase it to a request about doing it programatically specifically, as I believe that could still be a useful feature.

**Comment by TheNamlessGuy** [comment]
Actually, never mind. Didn't look at what the shortcut it generated actually did. Turns out you can already.
